src/theme/material/popup-confirmation.m.css

```diff
@@ -0,0 +1,42 @@
+.popupContainer {
```

Comment:
Could this use some of the mdc dialog styles? at the very least it should be using the appropriate elevation variable for the shadow.